drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/s3_zip_files_importer.rb

```diff
@@ -42,7 +44,7 @@ def self.mper
     end
 
     def run!
-      s3 = AwsS3.new(bucket_name: bucket_name)
+      s3 = remote_credential&.s3 || AwsS3.new(bucket_name: bucket_name)
```

Comment:
This is so weird, the credentials get set in the class method, then sorta kinda passed into the object.  Can we just set the credentials in the initializer?